internal/installutil/k8s/k8s.go

```diff
@@ -0,0 +1,82 @@
+package k8s
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/waypoint/internal/clierrors"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+type K8sConfig struct {
```

Comment:
I like the approach of having the K8sConfig here, since runner install and server install currently both have the same elements in their config structs! However, this has me thinking, do they (the server installer's and the runner installer's config) need to be the same, or could some of these elements be removed from either one?

Reply:
That's a good point, there is a lot of stuff in this struct that no longer needs to be here now that we've ripped out most of the k8s bits. And really its only for uninstalls only now that the rest of the installation is handled by helm. I'll remove the install specific configurations.

Reply:
Resolved in https://github.com/hashicorp/waypoint/pull/3354/commits/36e943e2457c4eac4a0cedcc2e2c79f82a5d0eb1